README.md

```diff
@@ -61,11 +61,11 @@ The exact version of Docker may be different than the above output.
 ### Getting the image BEFORE the workshop (highly recommended)
 
 1. Download the image file from [the latest release](https://github.com/ros-realtime/roscon-2023-realtime-workshop/releases/latest). The file is called `image.tar.gz`.
-2. Clone this repository.
-3. `cd` into this repository.
+2. Clone [roscon-2023-realtime-workshop](https://github.com/ros-realtime/roscon-2023-realtime-workshop) repository.
```

Comment:
We can actually just add `git clone --recursive https://github.com/ros-realtime/roscon-2023-realtime-workshop.git` as the command here. This will allow us to skip the `git submodule init && git submodule update` step below.

Reply:
resolved with 3005491